### ISSUE TITLE
about train

### ISSUE THREAD

**Issue by c-dada** [AdneneBoumessouer/Anomaly-Detection#30]
Hello，when I finish training,displays“Please invoke the Learner.lr_plot() method to visually inspect the loss plot to help identify the maximal learning rate associated with falling loss.”what shoud I do next.
      Looking forward to your reply.Thank you.

**Comment by AdneneBoumessouer** [comment]
Actually it wasn't training per se. It was just a simulation of training in order to determine the best learning rate, in which learning rates are being increased exponentially from a very small number untill the loss becomes too large. At the end of this process you are prompted to input the optimal learning rate. To do this, open the `lr_find_plot.png` image stored in the directory of your saved model and choose a suitable learning rate.

Example:
If you trained an mvtec2 model on the bottle dataset using SSIM loss, the image would be stored in: 
`Anomaly-Detection/saved_models/mvtec/bottle/mvtec2/SSIM/27-04-2020_16-08-12/lr_find_plot.png`
Then, you should choose a suitable learning rate corresponding to a falling loss.
Please read the corresponding section in [this post](https://www.pyimagesearch.com/2019/08/05/keras-learning-rate-finder/).

I am planning to do some modifications in the near future to make everything run automatically.

**Comment by c-dada** [comment]
Thank for your replay,I have run successfully for my own dataset,but the result is bad,mabye my settings have some wrong.I want to ask whether this work can mark the defect area.

**Comment by AdneneBoumessouer** [comment]
Not for the moment. I am planning on adding this fnuctionality in the near future. Thank you for your suggestion.
